### DISCUSSION TITLE
help!

### DISCUSSION THREAD

**Discussion by echo-escape** [zetaloop/ExplorerPatcher#33] [Q&A]
当我下了ASSET里的setup.exe文件，一点击之后电脑开始一直运行，导致我无法使用电脑，我想结束其运行，我该怎么做，但可以打开任务管理器
有办法解决吗，不然我只能重装系统了
万分感谢

**Comment by zetaloop** [comment]
安装不应该持续很长时间。如果桌面一直没有恢复，按快捷键 Ctrl + Shift + Esc 打开任务管理器，运行新任务 explorer 即可恢复桌面。

**Reply by zetaloop** [reply]
如果你需要结束setup.exe，可以在任务管理器里结束。
如果你需要卸载，可以参考主页提供的卸载方法。

**Reply by zetaloop** [reply]
按我的经验。安装程序基本上只是复制个文件，不应该这么大占用才对，你的电脑配置是否不太理想？之前我使用 i5-7200U 的笔记本，安装经常花费很长时间，而且会卡住导致桌面不恢复，更新了 i9-13900H 的笔记本后这些问题消失了。

**Reply by echo-escape** [reply]
感谢回复
使用的是i5-11400
虽然可以打开任务管理器，但无法通过上述方法，只能用CTRL+ALT+DEL 打开，并且由于其似乎一直运行，似乎总是处于"刷新”状态而导致无法使用
另外，你说得是否是任务管理器的explorer.exe，我已经重新运行，但只占用内存，不占用CPU
最后，在我一开始点击了setup后我的状态栏就已经变成了win10的样式
我是否需要像你说的等待上一段时间，复制完成就行了

**Reply by zetaloop** [reply]
文件管理器完全无法启动的恶性问题我曾遇到过，ExplorerPatcher可能和你的系统版本存在兼容问题，请你以管理员权限打开命令提示符，输入`del C:\Windows\dxgi.dll`来强制卸载。

由于ep采用修改系统组件的方式实现功能，不可避免地会降低稳定性和兼容性，你可以选择使用start11等其他仿win10风格开始菜单，他们的兼容性会更好，感谢尝试ExplorerPatcher。

如果您愿意的话，可以向软件原作者valinet汇报这个问题。不过，这也可能是某个奇怪的微软系统组件或者第三方软件导致的特殊问题，排查修复可能存在困难。

**Comment by echo-escape** [comment]
由于各种原因，我已经被迫花了五十块大洋进行重装系统
总而言之，还是感谢您的耐心回复

**Reply by echo-escape** [reply]
我听修电脑的说这是“中毒”，我也不知其是否正确，在这我也一并反馈给你

**Reply by zetaloop** [reply]
麻烦你了，本软件为开源软件，您可自行检查不包含病毒。不过由于软件兼容问题带来困扰为你说声抱歉。
